CMakeLists.txt

```diff
@@ -8,18 +8,24 @@ cmake_minimum_required(VERSION 3.14.6)
 
 cmake_policy(SET CMP0083 NEW)
 
+include(cmake/utilities.cmake)
```

Comment:
I am guessing these need to remain included before the `project` definition? Should we leave a comment/note about why these are included before the rest?

Reply:
Added one :)